<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pose â†’ Command (Embedded Video)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_both">
        <property name="text">
         <string>Both hands up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_Both_hands_up"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_left">
        <property name="text">
         <string>Left hand up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox_Left_hand_up"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_right">
        <property name="text">
         <string>Right hand up</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_Right_hand_up"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_idle">
        <property name="text">
         <string>Idle</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBox_Idle"/>
      </item>
      <item row="0" column="4" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_buttons">
        <item>
         <widget class="QPushButton" name="StartButton">
          <property name="text">
           <string>Start</string>
          </property>
          <property name="minimumWidth">
           <number>120</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CloseButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="text">
       <string>Video will appear here</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>360</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <widget class="QLabel" name="gestureLabel">
        <property name="text">
         <string>Gesture: -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actionLabel">
        <property name="text">
         <string>Action: -</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
